content/en/docs/started/workstation/minikf-gcp.md

```diff
@@ -2,16 +2,6 @@
 title = "Deploy using MiniKF on GCP"
 description = "Using Google Cloud Marketplace to deploy MiniKF (mini Kubeflow) on Google Cloud Platform (GCP)"
 weight = 40
-                    
-+++
```

Comment:
As per https://github.com/kubeflow/website/pull/2301 please add back this line to fix the deployment issue